.github/BOTMETA.yml

```diff
@@ -293,6 +293,7 @@ files:
   $modules/packaging/os/redhat_subscription.py: alikins kahowell
   $modules/packaging/os/rhn_channel.py: alikins
   $modules/packaging/os/rhn_register.py: $team_rhn
+  $modules/packaging/os/snap.py: angristan
```

Comment:
So you don't have to add modules in BOTMETA.yml if this information is already in the module.
The BOTMETA.yml is to extend the information that is on the modules, we don't need the same information twice.

Reply:
Oh, ok! Should I make another PR?